### ISSUE TITLE
Transparency of Modal Windows

### ISSUE THREAD

**Issue by nikmes** [ocornut/imgui#470]
Is there a way to control transparency of a modal window? Looks like below dose not allow to pass ALPHA value as an argument. 
if (ImGui::BeginPopupModal("File Operations", NULL, ImGuiWindowFlags_HorizontalScrollbar))
I tried to play with below:
ImGui::PushStyleVar(ImGuiStyleVar_Alpha, 0.f);
But didnt get desired results. Instead i got Modal window on a GREY background.
Thanks!

**Comment by nikmes** [comment]
![image](https://cloud.githubusercontent.com/assets/2520168/12089370/bb8f2dba-b2ec-11e5-8b0a-4e27b8e62d1a.png)

I don't want the Warning Window to have transparency at all.

**Comment by ocornut** [comment]
There is an inconsistency discussed in #337, you could alter the "style.WindowFillAlphaDefault" field and set it to 1.0.

This field may be removed and instead the value flattened across the different WindowBg colors.

**Comment by nikmes** [comment]
Thanks! I will try it out.

**Comment by ocornut** [comment]
Yes this should be the correct workaround. I am hesitating to add an enum for PushStyleVar() because this value should be removed eventually. See the update I just made to #337.

**Comment by nikmes** [comment]
Hi i checked the update. Wouldn't be easier just to allow the ImGui::BeginPopupModal to accept arguments like the ImGui::Begin("Grid Properties", opened, ImVec2(0, 0), 0.90f, ImGuiWindowFlags_NoScrollbar) ?

**Comment by ocornut** [comment]
No because that 'alpha' parameter is part of an obsolete Begin() entry point and is very rarely used. For a rarely used entry point it would make more sense to have a PushStyleVar(WindowFillAlpha, 1.0f) thing.
